lib/ziti-tunnel-cbs/ziti_hosting.c

```diff
@@ -314,7 +315,9 @@ static void on_hosted_udp_server_data(uv_udp_t* handle, ssize_t nread, const uv_
     struct hosted_io_ctx_s *io_ctx = handle->data;
     if (nread > 0) {
         io_ctx->in_wreqs++;
-        int zs = ziti_write(io_ctx->client, buf->base, nread, on_hosted_ziti_write, buf->base);
+        ZITI_LOG(TRACE, "sending %zd bytes service[%s] client[%s] server[%s]", nread, io_ctx->service->service_name,
```

Comment:
we have users that enable trace logging all the time for troubleshooting. i'd really rather us have some better way of enabling this sort of logging either dynamically or just at startup.

i'd feel better without this TRACE for now? happy to discuss if you want

Reply:
as you wish